Tribler/community/channel/__init__.py

```diff
@@ -0,0 +1,3 @@
+'''
+The channel package contains TTTODO: what does this do?
```

Comment:
The channel package contains a Dispersy community which is used to implement Channels. Channels are lists "favorite" .torrents create by one or more users.